nalgebra-lapack/tests/linalg/generalized_eigenvalues.rs

```diff
@@ -0,0 +1,74 @@
+use na::dimension::{Const, Dynamic};
+use na::{DMatrix, EuclideanNorm, Norm, OMatrix};
+use nl::GE;
+use num_complex::Complex;
+use simba::scalar::ComplexField;
+use std::cmp;
+
+use crate::proptest::*;
+use proptest::{prop_assert, proptest};
+
+proptest! {
+    #[test]
+    fn ge(n in PROPTEST_MATRIX_DIM) {
+        let n = cmp::max(1, cmp::min(n, 10));
+        let a = DMatrix::<f64>::new_random(n, n);
+        let b = DMatrix::<f64>::new_random(n, n);
```

Comment:
This kind of generation is fairly problematic, since `proptest` does not know about the random generation, and the test is moreover non-deterministic and non-repeatable. In particular, if the test fails you cannot rerun it again with the same input (`proptest` has a mechanism for storing failing input seeds to disk so that if you re-run the test it will try the list of known failing seeds).